### ISSUE TITLE
build error b

### ISSUE THREAD

**Issue by JMPHCD** [3ct0s/eclipse-grabber#26]
![image](https://user-images.githubusercontent.com/99404569/153407842-2f9e9b41-ff4e-446b-9d19-77e7ec482cb8.png)
i need to fix this

**Comment by 3ct0s** [comment]
This is the first time I am seeing this error, can you please tell me whether you have run the setup installation without any issues?

**Comment by JMPHCD** [comment]
yes ive runned it and then i runned the file in my pc to see if it works but i forgot i have real time protection on so it deleted the dist file so what i did was redo it again. i watched the video on how to do it and i typed all the commands and then when it comes to the last one it said that error

**Comment by JMPHCD** [comment]
no i meant the file i the folder didnt get deleted

**Comment by JMPHCD** [comment]
i tried deleting the dist folder and rerunning the command again still the same error
